ui/packages/catalog-ui-search/src/main/webapp/js/router/routes.js

```diff
@@ -59,4 +59,20 @@
       classes: 'is-bold',
     },
   },
+  openForms: {
+    patterns: ['forms(/)'],
+    component: 'component/tabs/search-form/tabs.search-form.view',
+    menu: {
+      text: 'Search Forms',
+      classes: 'is-bold',
+    },
+  },
+  newForm: {
+    patterns: ['forms/:id'],
+    component: 'component/search-form-editor/search-form-editor.view',
+    menu: {
+      text: 'New Search Form',
```

Comment:
Should this always be `New Search Form`? Should it be different if you are editing an existing form?

Reply:
Good catch. I'll change it to `Search Form Editor`.